backend/core/package.json

```diff
@@ -19,13 +19,10 @@
     "db:drop": "psql -c 'DROP DATABASE IF EXISTS bloom_detroit;'",
     "db:create": "psql -c 'CREATE DATABASE bloom_detroit;'",
     "db:add-uuid-extension": "psql -d bloom_detroit -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
-    "db:seed": "ts-node src/seeder/seed.ts",
```

Comment:
with this change the only reference to `seed.ts` is the `test:db:setup` task. Should we also remove it from there as well?

Reply:
As an addition to that, should we also remove the seed.ts file altogether?